Directory.Packages.props

```
@@ -36,13 +39,21 @@
     <PackageVersion Include="Microsoft.AspNetCore.WebHooks.Receivers.GitHub" Version="1.0.0-preview2-final" />
     <PackageVersion Include="Microsoft.Azure.Kusto.Ingest" Version="12.2.1" />
     <PackageVersion Include="Microsoft.Azure.WebJobs.Extensions.Storage" Version="5.0.1" />
+    <PackageVersion Include="Microsoft.Bot.Builder" Version="4.22.0" />
+    <PackageVersion Include="Microsoft.Bot.Builder.AI.Luis" Version="4.22.0" />
+    <PackageVersion Include="Microsoft.Bot.Builder.ApplicationInsights" Version="4.22.0" />
+    <PackageVersion Include="Microsoft.Bot.Builder.Dialogs" Version="4.22.0" />
+    <PackageVersion Include="Microsoft.Bot.Builder.Integration.ApplicationInsights.Core" Version="4.22.0" />
+    <PackageVersion Include="Microsoft.Bot.Builder.Integration.AspNet.Core" Version="4.22.0" />
+	<PackageVersion Include="Microsoft.Bot.Builder.Testing" Version="4.22.0" />
+    <PackageVersion Include="Microsoft.Bot.Connector" Version="4.22.0" />
     <PackageVersion Include="Microsoft.Build" Version="17.1.0" />
     <PackageVersion Include="Microsoft.Build.Framework" Version="17.1.0" />
     <PackageVersion Include="Microsoft.Build.Tasks.Core" Version="17.1.0" />
     <PackageVersion Include="Microsoft.Build.Utilities.Core" Version="17.1.0" />
     <PackageVersion Include="Microsoft.Data.SqlClient" Version="2.1.7" />
     <PackageVersion Include="Microsoft.DncEng.CommandLineLib" Version="$(MicrosoftDncEngCommandLineLibVersion)" />
-    <PackageVersion Include="Microsoft.DncEng.Configuration.Extensions" Version="$(MicrosoftDncEngConfigurationExtensionsVersion)" />
+	<PackageVersion Include="Microsoft.DncEng.Configuration.Extensions" Version="$(MicrosoftDncEngConfigurationExtensionsVersion)" />
```

```suggestion
    <PackageVersion Include="Microsoft.DncEng.Configuration.Extensions" Version="$(MicrosoftDncEngConfigurationExtensionsVersion)" />
```
tabs and spaces oh my!

---

I'm not really sure why it looks like that on github bc it looks normal on my side :(

---

Oh wait, are they supposed to be tabbed in?